addons/mail/static/src/composer/suggestion_service.js

```diff
@@ -216,9 +216,9 @@ export class SuggestionService {
             if (!isAInternalUser && isBInternalUser) {
                 return 1;
             }
-            if (thread?.channel) {
-                const isMember1 = thread.channel.channelMembers[0][1].includes(p1);
-                const isMember2 = thread.channel.channelMembers[0][1].includes(p2);
+            if (thread.model === "discuss.channel") {
```

Comment:
This should use `thread.isChannel`. The getter should be the way to test that a thread is a discuss channel, we don't really want to have so much hard coded condition every where in the code.

Reply:
`isChannel` is the same as `.model === "discuss.channel"`.
It's best to not introduce alias in the code and keep it simple.

Maybe we'd prefer to use something more like `isChannel` in the future, by replacing all `.model === "discuss.channel"`, but this is definitely out-of-scope of this PR.